AsyncDisplayKit/ASLayoutElementInspectorCell.m

```diff
@@ -197,23 +222,23 @@ - (void)buttonTapped:(ASButtonNode *)sender
   BOOL selfIsEditing = (self == __currentlyOpenedCell);
   [__currentlyOpenedCell endEditingValue];
   if (selfIsEditing) {
+    sender.selected = NO;
     return;
   }
   
 //  NSUInteger currentAlignSelfValue;
 //  NSUInteger nextAlignSelfValue;
 //  CGFloat    newValue;
-  
+
+  sender.selected = !sender.selected;
```

Comment:
Should this actually be toggled for the properties that aren't treated like booleans, like AlignSelf?  This code was written when flexGrow and flexShrink were actual BOOLs, but even with the 0.0 / 1.0 implementation they make sense to toggle in the "on/off" sort of way.  We could make UISwitches as an alternative idea.